mappings/net/minecraft/text/TextColor.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_5251 net/minecraft/text/TextColor
+	COMMENT Represents an RGB color of a {@link Text}.
+	COMMENT
+	COMMENT <p>This is immutable, and part of a {@link Style}.</p>
+	COMMENT
+	COMMENT @see Style
+	FIELD field_24362 FORMATTING_TO_COLOR Ljava/util/Map;
+	FIELD field_24363 BY_NAME Ljava/util/Map;
+	FIELD field_24364 rgb I
+	FIELD field_24365 name Ljava/lang/String;
+	METHOD <init> (I)V
+		ARG 1 rgb
+	METHOD <init> (ILjava/lang/String;)V
+		ARG 1 rgb
+		ARG 2 name
+	METHOD method_27716 getRgb ()I
+		COMMENT Gets the RGB value of this color.
+		COMMENT
+		COMMENT <p>The red bits can be obtained by {@code (rgb >> 16) & 0xFF}, green bits
+		COMMENT by {@code (rgb >> 8) & 0xFF}, blue bits by {@code rgb & 0xFF}.</p>
+	METHOD method_27717 fromRgb (I)Lnet/minecraft/class_5251;
+		COMMENT Obtains a text color from an RGB value.
+		ARG 0 rgb
+			COMMENT the RGB color
+	METHOD method_27718 fromFormatting (Lnet/minecraft/class_124;)Lnet/minecraft/class_5251;
+		COMMENT Obtains a text color from a formatting.
+		ARG 0 formatting
+			COMMENT the formatting
+	METHOD method_27719 parse (Ljava/lang/String;)Lnet/minecraft/class_5251;
+		COMMENT Parses a color by its name.
+		ARG 0 name
+			COMMENT the name
+	METHOD method_27721 getName ()Ljava/lang/String;
+		COMMENT Gets the name of this color, used for converting the color to JSON format.
+	METHOD method_27723 getRgbName ()Ljava/lang/String;
```

Comment:
RGB name?

Reply:
yep, private method, used only by rgb-backed color, returns a string like `#abcdef`

Reply:
I think `getRgbString` would be better since it's not really a name.

Reply:
isn't that a hex code? so couldn't it be like ``toHex`` or something?

Reply:
It is hex prefixed with `#`

Reply:
```suggestion
	METHOD method_27723 getHexCode ()Ljava/lang/String;
```